axelor-stock/src/main/java/com/axelor/apps/stock/service/StockMoveLineServiceImpl.java

```diff
@@ -311,6 +311,8 @@ public StockMoveLine createStockMoveLine(
       stockMoveLine.setProductTypeSelect(product.getProductTypeSelect());
     }
 
+    stockMoveLine.setSequence(stockMove.getStockMoveLineList().size() * 10);
+
```

Comment:
don't multiply by 10 because it will be changed when user move lines manually from o2m.